Documentation/Signoffs/PLC.md

```diff
@@ -0,0 +1,80 @@
+## Subsystem Function 
+
+The subsystem's role in our design is to control all electronic hardware (locks, LEDs, etc) as well as the motor rotating the platforms. It will also have a communication stream with the microcomputer to inform it when a board is checked out or restocked. 
+
+## Constraints 
+
+**The PLC:** 
+
+* **Must have at least 6 Digital Input pins**
+
+^ These are for the 3 door locks and sensors in order to be read by the PLC.
+
+* **Must have at least 6 Digital Output pins**
+
+^ These are also for the 3 door lock and sensors in order to be commanded by the PLC.
+
+* **Must have at least 3 24VDC Outputs**
+
+^ These are going to be used for the motor driver.
+
+From this, the input module must have at least 6 ports, and the output module must have at least 9 ports 
+
+* **Must be able to fit in a 1 square-foot area**
+
+^ This is for clearance in the area where we are going to house all of our controllers and electronics, aside the motor. 
+
+* **Will be in accordance with IEC-61131-2(6.4.4.3) which states:**
+
+"When the short circuit (for a voltage output) or the open circuit (for a current output) is realized, no physical damage or abnormal phenomenon shall be detected." 
+
+This will be accomplished by having the GFCI in between our hardware and the power supply. 
+
+^ Source: http://rpa.energy.mn/wp-content/uploads/2017/03/IEC-61131-2-Programmable-controllers-Equipment-Requirements-and-Tests.pdf 
+
+* **Must be able to communicate with our PC
```

Comment:
Needs to be discussed here. 

Saying ethernet plus software like Allen Bradley tells me that you do not understand your communication needs (which is ok). But then you need to investigate and ask questions so that you have reasonable certainty. Designs aren't guesses.